docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -100,128 +99,226 @@ In the case of errors, include human-readable error messages in the response pay
-POST https://example.com/shipping/carrier/connection
+After submitting your app, you will receive the a carrier ID. Both single-carrier and multi-carrier shipping providers receive one `carrier_id`.
+
+### What should you document for API users?
+ 
+We recommend that you document your carrier ID and configuration option codes for API users who wish to use your carrier. For example, API users will specify the `carrier_id` and `code` for connections options when they [connect your carrier to their store](#how-your-app-will-be-connected-to-a-store). API users specify the `code` for settings options when they define your carrier's shipping method for their shipping zones.  
+
+## What's next?
+
+### How your app will be connected to a store
+
+Once a merchant installs your app on their store, merchants and API users can connect your carrier to their store by using the connection options that you provided when you submitted your app.    
+
+#### How merchants will use your app
+A merchant can navigate to the Shipping Manager UI to input carrier connection options. The UI displays the connection option's `label` that you provided when you submitted your app. The following figure is an example of how connection options are shown to merchants in the control panel:
+
+![Connect Carrier via UI](https://storage.googleapis.com/bigcommerce-production-dev-center/images/connection%20options%20example.png 'Connection options in Shipping Manager') 
+
+After connecting your carrier, a merchant can define and enable a real-time shipping method for your carrier app in one or more shipping zones. 
+
+#### How API users will use your app
+API users can connect your carrier to the store by using the [Create a carrier connection](/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint. In the request, API users will send the carrier ID that you received when you signed up, as well as values for your app's connection options. Specifically, API users will specify the `code` for each connection option as a property under the `connection` object when they connect to your carrier, for example:    
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/shipping/carrier/connection
 X-Auth-Token: {{ACCESS_TOKEN}}
 Content-Type: application/json
 Accept: application/json
 
-{
-  "carrier_id": "carrier_33",
+{	
+  "carrier_id" : "endicia",
   "connection": {
-    "key": "userKey",
-    "account_number": "userAccountNumber"
+      "account_id" : "example_id",
+      "pass_phrase" : "example_passphrase"
   }
 }
 ```
-&nbsp;
-```http title="Example carrier connection request without connection settings" lineNumbers
-POST https://example.com/shipping/carrier/connection
-X-Auth-Token: {{ACCESS_TOKEN}}
-Content-Type: application/json
-Accept: application/json
 
-{
-  "carrier_id": "carrier_33",
-  "connection": {}
-}
-```
+<!-- theme:info -->
+> #### Note

```

Comment:
```suggestion
> #### Document connection options

```
if you're specific with the headings, we can use anchor links to callouts in other docs.